test/clj/doit/todo_list/db_test.clj

```diff
@@ -17,3 +17,20 @@
           response (todo-list-db/create! {:name name})]
       (is (= (:name response) name))
       (is (= (set (keys response)) #{:name :id})))))
+
```

Comment:
A db level test for deleting a todo list seems to be missing here.

Reply:
Agree, will add that.